_episodes/06-files.md

```diff
@@ -12,9 +12,8 @@ keypoints:
 - "Use `*` in a pattern to match zero or more characters, and `?` to match any single character."
 ---
 
-We now have almost everything we need to process all our data files.
-The only thing that's missing is a library with a rather unpleasant name:
-
+As a final piece to processing our inflammation data, we need a way to get a list of all the files in our data directory whose names start with inflammation- and end with .csv.
```

Comment:
Thanks @codingS3b , I think this text works. I am including a formatting suggestion here that should make it more consistent with our style trends (Keeping lines under 100 characters and using back ticks to set off certain text).
```suggestion
As a final piece to processing our inflammation data, we need a way to get a list of all the files
in our `data` directory whose names start with `inflammation-` and end with `.csv`.
```